docker/docker.mk

```diff
@@ -16,10 +16,18 @@
 DOCKER:=docker
 DOCKERFILE_DIR?=./docker
 
+# If set to true, only the `dapr` image will be built and pushed to the registry.
+# This is a "kitchen sink" image that contains all the components.
+# The Helm charts will also be configured to use this image.
+# This is useful for faster development and testing experience.
+USE_DEV_IMAGE?=false
```

Comment:
Sorry but can this have a name that better relates to the intent?

Reply:
Any suggestions? `USE_DEV_IMAGE` sounded ok because the intent is to reduce development time by building/creating a single image.

Reply:
How about `NO_SPLIT_IMAGES` or `SINGLE_IMAGES` (or `KITCHEN_SINK_IMAGE`)?

Reply:
Good ideas, shall we go ahead with `USE_SINGLE_IMAGE`? @artursouza, thoughts?

Reply:
If it is true, only build `RELEASE_NAME` image, isn't it?

Reply:
Updated it!

Reply:
Need @artursouza to merge since it is a requested change.